frontend/views/components/modal/Modal.vue

```diff
@@ -70,19 +73,15 @@ export default {
     activeSubcontent () {
       return this.subcontent[this.subcontent.length - 1]
     },
-    initializeModals () {
-      const modal = this.$route.query.modal
-      if (modal) this.openModal(modal)
-      const subcontent = this.$route.query.subcontent
-      if (subcontent) this.openModal(subcontent)
-    },
     updateUrl () {
       if (this.content) {
         this.$router.push({
           query: {
             ...this.$route.query,
-            modal: this.content,
-            subcontent: this.activeSubcontent()
+            ...{
+              modal: this.content,
+              subcontent: this.subcontent.length ? this.subcontent.join('+') : undefined
+            }
```

Comment:
@sandrina-p removed the extra braces `{ }` around `modal` and `subcontent`, is there a reason they were added back?